CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
     Click to see more.
   </summary>
 
+### Major
+
+- Box [Breaking]: Removed support for `shape` prop values `roundedTop`, `roundedRight`, `roundedBottom`, and `roundedLeft` _Has codemod_ (#657)
```

Comment:
@jennyscript Instead of `Has codemod`, can we give people the command they need to run for each one of these?

```sh
cd gestalt
yarn run codemod --parser=flow -t=packages/gestalt-codemods/0.108.0-0.109.0/convert-font-weight.js ~/code/repo
```